lib/resque/scheduler/job.rb

```diff
@@ -0,0 +1,40 @@
+# vim:fileencoding=utf-8
+
+module Resque
+  module Scheduler
+    module Job
+      class << self
+        def included(base)
+          base.extend ClassMethods
+        end
+      end
+
+      module ClassMethods
+        def cron(value = nil)
```

Comment:
If feels like it could be cleaner to have a single class method `resque_schedule` which has a bunch of optional args (`cron:`, `every:`, etc). I don't love the idea of adding a bunch of methods that aren't clearly ties to resque scheduler (i.e. `description` could easily be a method users of this gem have defined on their class).

What do you think?

Reply:
Yes code looks cleaner this way 😄